### PR TITLE
Feature/databricks kainos name change

### PR DESCRIPTION
**Pull request by ShannonHolgate**
Changing package name to ensure we do not get conflicts in CDH cluster

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -30,6 +27,14 @@ matrix:
     - jdk: openjdk7
       scala: 2.11.7
       env: TEST_HADOOP_VERSION="2.2.0" TEST_SPARK_VERSION="1.5.0"
+    # Spark 1.6.0 and Scala 2.10
```
Comment (adamf-kainos) [inline .travis.yml]: Comment says Scala 2.10, but your actual version is 2.11.7

File: build.sbt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name := "spark-avro"
 
-organization := "com.databricks"
+organization := "com.kainos"
 
 scalaVersion := "2.10.5"
```
Comment (ghost) [inline build.sbt:5]: in the travis.yml you reference scala version 2.10.4, here it is 2.10.5. Not sure the mismatch matters, but might want to keep as much consistency as possible here

**Review by ghost — CHANGES_REQUESTED** [review]
Could you just leave a quick comment about the comment around the scala version mismatch and I can approve 👍

File: build.sbt (hunk 2 above)
Comment (ShannonHolgate) [reply build.sbt:5]: Good spot. I don't believe this matters as they are both 2.10. I'll change this one to keep it consistent

**Review by ghost — APPROVED** [review]
Happy to merge

**Pull request merged**